test_menu.js

```diff
@@ -0,0 +1,90 @@
+//Tests all menu blocks for expected behaviour.
+var url = "https://neontabs.neontribe.org/latest/hr/"
+var base_url = "https://neontabs.neontribe.org"
+var link_count = 0;
+var tests = link_count + 10;
+
+casper.start(casper.cli.get("target"), function(){
```

Comment:
we'll want to pick a consistent argument name for these. I use "home" in all of my scripts.
see comments from https://github.com/neontribe/hr-tests/pull/1